spec/gar_spec.lua

```diff
@@ -516,6 +532,12 @@ describe("gar", function()
 					},
 				},
 			}
+			local expectedSettings = utils.deepCopy(testSettings)
+			expectedSettings.allowedDelegatesLookup = {
+				["test-allowlisted-delegator-address-number-1"] = true,
+				["test-allowlisted-delegator-address-number-2"] = true,
+				[stubRandomAddress] = true,
```

Comment:
why is this in the list?

Reply:
Permission to delegate is indicated by either:
- presence in the allow list
- holding a delegated stake balance
When your stake is exited from a gateway, your permission is preserved by re-entering your in the allow list. In this case, since the delegate wasn't explicitly removed from the allow list, they were moved back into the allow list when their delegated stake was exited.

Reply:
got it, makes sense.